content/docs/user-guide/setup-google-drive-remote.md

```diff
@@ -1,9 +1,61 @@
 # Setup a Google Drive DVC Remote
 
-Follow this guide to setup Google Drive as your <abbr>DVC project</abbr>'s
-[remote storage](/doc/command-reference/remote).
+In this guide we explain the existing alternatives to setup Google Drive
+[remote storage](/doc/command-reference/remote) for your <abbr>DVC
+project</abbr>, along with the different benefits each one brings.
 
-## Configure a Google Cloud project
+Note that to start using a GDrive remote, you only need to add it with a valid
```

Comment:
see above, I think we need a quick setup or basic setup or something

this whole doc should be self-contained, so should include `dvc remote add` in the very beginning.

Reply:
I just added a basic example under this paragraph now, would that be enough? Our user guides aren't typically super self-contained or fully reproducible like tutorials though, we do link to the cmd refs.

Reply:
yes, it's enough .. I was thinking about being more explicit and making a title like "Quick start" or "Basic setup"